ZenPacks/zenoss/ZenPackLib/lib/spec/Spec.py

```diff
@@ -171,6 +174,43 @@ def __str__(self):
 
         return "{}({})".format(self.__class__.__name__, ' - '.join(parts))
 
+    @property
+    def order(self):
+        """order attribute used by ClassSpec, ClassRelationSpec, ClassPropertySpec"""
+        return self._order
+
+    @order.setter
+    def order(self, value):
+        """order should be an integer between 1 and 100"""
+        # deal with legacy cases
+        if isinstance(value, int):
+            self._order = int(max(0, min(100, value)))
+        else:
+            self._order = value
+
+    @property
+    def normalize_order(self):
+        """Return True if order should be renormalized"""
+        return isinstance(self.order, float)
+
+    def normalize_children(self, items):
+        """Return True if order should be normalized"""
+        return any(x.normalize_order for x in items)
+
+    def normalize_child_order(self, items):
+        """Deal with legacy order parameters given as float"""
+        if self.normalize_children(items):
```

Comment:
Given that normalize_children is a single-line method, and this is the only place it's used, you may want to inline the its code then eliminate the normalize_children method.